system/Helpers/html_helper.php

```diff
@@ -184,6 +192,12 @@ function img($src = '', bool $indexPage = false, $attributes = ''): string
 			}
 		}
 
+		// prevent passing "alt" to stringify_attributes
+		if(is_array($attributes) && isset($attributes['alt']))
```

Comment:
If we are grabbing this value and shoving it into $src, is there a reason we cannot remove it there, and not do a separate check here?

Reply:
Yes there is a reason to do it separately . 
It $src['alt'] exists and $attributes['alt'] exists then funcs will produce: <img scr="img.jpg" alt="from alt" alt="from attributes" />

When I tested it first unset was in line 164... but there is that special case as I mentioned above.